packages/extension-ui/src/Popup/CreateAccount/index.tsx

```diff
@@ -81,9 +89,15 @@ function CreateAccount ({ className }: Props): React.ReactElement {
                 <Dropdown
                   className={className}
                   label={t<string>('Network')}
-                  onChange={setGenesis}
+                  onChange={(newGenesisHash: string) => {
+                    if (getChainName(newGenesisHash) === 'Moonbase Alpha') { // TODO: use list
+                      setType('ethereum');
+                    }
+
+                    setGenesis(newGenesisHash);
+                  }}
```

Comment:
```suggestion
                  onChange={(newGenesisHash: string) => {
                    if (getChainName(newGenesisHash) === 'Moonbase Alpha') { // TODO: use list
                      setType('ethereum');
                    }

                    setGenesis(newGenesisHash);
                  }}
```

We want this function defined inside the component with a `useCallback` definition. At this point it will be re-created each render, with the `useCallback` it will be crated once.

Reply:
@jacogr Thank you for reviewing.
I tried to quickly address your comment as Antoine is not available this week.
Let me know if there is more to do before you can merge it.

Reply:
Thank you.